README.md

```diff
@@ -197,7 +197,9 @@ func main() {
         wavefront.BatchSize(20000), // Send batches of 20,000.
         wavefront.FlushIntervalSeconds(5), // Flush every 5 seconds.
         wavefront.MetricsPort(4321), // Use port 4321 for metrics.
-        wavefront.TracesPort(40001), // Use port 40001 for traces.
+        wavefront.TracesPort(40001), // Use port 40001 for traces. 
```

Comment:
We moved all the example code within the source code so that it is compiled as part of the test suite and so that this example code shows up on pkg.go.dev. Can you please resync this PR before I review?

Reply:
Yes, done!